enslig-forsorger-test/src/main/kotlin/no/nav/familie/kontrakter/ef/søknad/TestsøknadBuilder.kt

```diff
@@ -730,7 +730,10 @@ class TestsøknadBuilder private constructor(
                             )
                         )
                     )
-                )
+                ),
+                semesteravgift = Søknadsfelt("Semesteravgift", 30000.toDouble()),
+                studieavgift = Søknadsfelt("Studieavgift", Double.NaN),
+                eksamensgebyr = Søknadsfelt("Semesteravgift" ,0.toDouble()),
```

Comment:
Er det riktig å kalle eksamensgebyr for Semesteravgift?

Reply:
Nei, fikset [6fe89ad](https://github.com/navikt/familie-kontrakter/pull/575/commits/6fe89ad4b5c78981affca63339df742b58cb2461)